spec/rubocop/cop/gemspec/require_mfa_spec.rb

```diff
@@ -0,0 +1,175 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Gemspec::RequireMFA, :config do
+  context 'when the gemspec is blank' do
+    it 'registers an offense but does not correct' do
+      expect_offense(<<~RUBY, 'my.gemspec')
+        ^{} `metadata['rubygems_mfa_required']` must be set to `'true'`.
```

Comment:
If blank, this cop will be able to skip the process (In other words, it would be `expect_no_offenses`). Because there are many other things missing 😅

Reply:
Removed.